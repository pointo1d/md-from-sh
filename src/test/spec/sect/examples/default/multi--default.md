# %FNAME%

---

# Author

%AUTHOR%

---

# Date

%DATE%

---

END OF FILE
