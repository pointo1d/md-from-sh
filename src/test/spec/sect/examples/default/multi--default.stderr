WARNING - Empty section: 'File' - line 1 (default used)
WARNING - Empty section: 'Author' - line 4 (default used)
WARNING - Empty section: 'Date' - line 7 (default used)
